base_layer/wallet_ffi/build.config

```diff
@@ -0,0 +1,7 @@
+BUILD_ANDROID=0 for false, 1 for true
```

Comment:
I propose renaming this file `build_sample.config` and adding `build.config` to `.gitignore` so that devs can set up their config file as they wish without interfering with the VCS.

Reply:
You might want to pre-populate the paths with sane defaults where it makes sense

Reply:
Agreed, will make the change.